astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -194,6 +194,116 @@ def test_cirs_to_hadec():
     assert_allclose(cirs.dec, cirs3.dec)
 
 
+def test_itrs_topo_to_altaz_with_refraction():
+
+    loc = EarthLocation(lat=0*u.deg, lon=0*u.deg, height=0*u.m)
+    usph = golden_spiral_grid(200)
+    dist = np.linspace(1., 1000.0, len(usph)) * u.au
+    icrs = ICRS(ra=usph.lon, dec=usph.lat, distance=dist)
+    altaz_frame1 = AltAz(obstime = 'J2000', location=loc)
+    altaz_frame2 = AltAz(obstime = 'J2000', location=loc, pressure=1000.0 * u.hPa,
+                         relative_humidity=0.5)
+    itrs_frame = ITRS(location=loc)
+
+    #Normal route
```

Comment:
Nitpick, but PEP8 likes spaces after `#` and full stops at end of comments (treat as sentences)

Reply:
To be sure, are you asking me to add periods to all of my comments in spite of the fact that most comments that I see elsewhere don't include them?

Reply:
It was more the space after `#`, but as I said, I was really nitpicking here!

Reply:
OK, I will add the spaces, but neglect the punctuation then.